style/ember/README.md

```diff
@@ -2,6 +2,7 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+* Don't use {{partial}} helper. Use a component with only a template.
```

Comment:
I assumed partial was deprecated, but it seems like it isn't. Will it be deprecated once routable components drop?